frontend/src/main.ts

```diff
@@ -4,31 +4,28 @@ import 'vue-material/dist/vue-material.css';
 import VueMaterial from 'vue-material';
 
 import Vue from 'vue';
-import App from './App';
+import App from './App.vue';
 import router from './router';
 import API from './api';
 import { Client } from '../../getline.ts';
+import { VueConstructor } from 'vue/types/vue';
+import registerPurpleTheme from './theme';
 
 Vue.use(VueMaterial);
 Vue.use(VueMaterial.MdIcon);
 
-
 setTimeout(async () => {
   // setTimeout because we are waiting for metamask extenion to inject the web3 object
   API.init(Client);
-  const myAddress = window.web3.eth.accounts[0];
-  console.log(await API.instance().getLoansByOwner(myAddress));  // eslint-disable-line
+  // const myAddress = window.web3.eth.accounts[0];
```

Comment:
Don't commit commented out code.

Also, please don't use web3 directly - the client library has a method to get the local address.